examples/MicroserviceExample/WebApi/Controllers/SendMessageController.cs

```diff
@@ -10,12 +10,10 @@ namespace WebApi.Controllers;
 [Route("[controller]")]
 public class SendMessageController : ControllerBase
 {
-    private readonly ILogger<SendMessageController> logger;
     private readonly MessageSender messageSender;
 
-    public SendMessageController(ILogger<SendMessageController> logger, MessageSender messageSender)
+    public SendMessageController(MessageSender messageSender)
     {
-        this.logger = logger;
```

Comment:
This needs a closer review. Since it's an example project it might have been the intention to demonstrate logs.

Reply:
can u point me to the online docs where this sample is referenced?

Reply:
This example has only trace examples, so it’s fine to remove it.